tools/ci/download_bin.sh

```diff
@@ -57,40 +61,73 @@ download_artifact_linux()
         echo "CircleCI build_num:${build_num} can't find artifacts."
         exit 1
     fi
-    echo -e "\033[32mDownloading binary from ${link} \033[0m"
-    cd ${output_dir} && curl -LO ${link} && tar -zxf fisco*.tar.gz && rm fisco*.tar.gz
+    LOG_INFO "Downloading binary from ${link} "
+    curl -#LO ${link} && tar -zxf fisco*.tar.gz && rm fisco*.tar.gz
     result=$?
-    if [[ "${result}" != "0" ]];then echo  -e "\033[31mDownload failed, please try again\033[0m" && exit 1;fi
-    echo -e "\033[32mFinished. Please check ${output_dir}\033[0m"
+    if [[ "${result}" != "0" ]];then LOG_ERROR "Download failed, please try again" && exit 1;fi
 
 }
 
 download_artifact_macOS(){
     echo "unsupported for now."
     exit 1
+    local branch="$1"
     # TODO: https://developer.github.com/v3/actions/artifacts/#download-an-artifact
-    local workflow_artifacts_url="$(curl https://api.github.com/repos/FISCO-BCOS/FISCO-BCOS/actions/runs\?branch\=${branch}\&status\=success\&event\=push | grep artifacts_url | head -n 1 | cut -d \" -f 4)"
+    local workflow_artifacts_url="$(curl https://api.github.com/repos/${project}/actions/runs\?branch\=${branch}\&status\=success\&event\=push | grep artifacts_url | head -n 1 | cut -d \" -f 4)"
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5859193)